draft-ietf-oauth-selective-disclosure-jwt.md

```diff
@@ -914,6 +914,51 @@ The work on this draft was started at OAuth Security Workshop 2022 in Trondheim,
 
 TBD
 
+## Media Type Registration
+
+This section requests registration of the "application/sd-jwt" media type [@RFC2046] in
+the "Media Types" registry [@IANA.MediaTypes] in the manner described
+in [@RFC6838], which can be used to indicate that the content is an SD-JWT.
+
+* Type name: application
+* Subtype name: sd-jwt
+* Required parameters: n/a
+* Optional parameters: n/a
+* Encoding considerations: binary; application/sd-jwt values are a series of base64url-encoded values (some of which may be the empty string) separated by period ('.') or tilde ('~') characters.
+* Security considerations: See the Security Considerations section of [[ this specification ]].
```

Comment:
imo, you should also reference security considerations of RFC7519 here.

Reply:
Makes sense. I'll add that. And maybe the JWT BCP RFC8725 too.

Reply:
done